custom_components/ocpp/api.py

```diff
@@ -716,10 +716,7 @@ def process_phases(self, data):
                     + value[Phase.l2.value]
                     + value[Phase.l3.value]
                 )
-                if sum > 0:
-                    self._metrics[metric] = round(sum / 3, 1)
-                else:
-                    self._metrics[metric] = round(sum, 1)
+            self._metrics[metric] = round(sum, 1)
```

Comment:
Why round here? I assume it is for presentation purposes only, so maybe better do that in the sensor entity? 
We could extend the upcoming Metric class to have a format, besides value and unit.

Reply:
Yes, only reason is presentation. Agree it would be better to handle format in the sensor. If sensor formats then other `round` functions can be removed from api also.